.github/workflows/mkosi.yml

```diff
@@ -153,8 +153,8 @@ jobs:
           [Build]
           UseSubvolumes=yes
           ToolsTree=default
-          ToolsTreeDistribution=fedora
-          ToolsTreeRelease=rawhide
+          ToolsTreeDistribution=${{ matrix.distro }}
+          ToolsTreeRelease=${{ matrix.release }}
```

Comment:
Does it now work? When I asked previously, you said it does not work.

Reply:
It doesn't work for all distros, but I added the change to figure out what doesn't work, I'll drop it again to get CI to pass

Reply:
So opensuse has a weird build failure that I can't look into until https://build.opensuse.org/request/show/1232687 is fixed. CentOS 9/10 don't have `btrfs` which we need to make the tests not go over the limited disk size. And on Ubuntu `meson` is too old and doesn't have `--max-lines`.